lib/Reader/reader.cpp

```diff
@@ -7342,6 +7355,10 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
       break;
 
     case ReaderBaseNS::CEE_RET: {
+      // Insert a nop for debug and set return's IL offset to signify
+      // that it is part of the epilogue
+      nop();
+      setDebugLocation(ICorDebugInfo::EPILOG, false);
```

Comment:
Nit: please life the constant `false` into a local similar to the other call sites: `const bool IsCall = false`.